osu.Game/Input/Bindings/GlobalActionContainer.cs

```diff
@@ -186,6 +190,12 @@ public static IEnumerable<GlobalAction> GetGlobalActionsFor(GlobalActionCategory
             new KeyBinding(InputKey.Minus, GlobalAction.DecreaseOffset),
         };
 
+        private static IEnumerable<KeyBinding> multiplayerKeyBindings => new[]
+        {
+            new KeyBinding(new[] { InputKey.Control, InputKey.R }, GlobalAction.MultiplayerReady),
+            new KeyBinding(new[] { InputKey.Control, InputKey.S }, GlobalAction.MultiplayerSpectate),
```

Comment:
These seem kinda weird / arbitrary. I'd propose something like ctrl/shift-enter for these or something. Something that has more weight behind it, these are supposed to be primary actions for this screen.

Reply:
I'm not sure what to do about this, I just picked whatever made sense for me when I made the PR, but I wasn't actually sure if these keybindings were good or not. Ctrl-enter seems good for the ready button, but I still don't really know what to choose for the spectate button (maybe leave it unbinded by default ?).

Reply:
interested in more opinions from @ppy/team-client on this one